### PR TITLE
EZP-30882: As a developer I want to use custom Field Type as Content thumbnail

### PR DESCRIPTION
**Pull request by Nattfarinn**
# Depends on: https://github.com/ezsystems/ezpublish-kernel/pull/2868

### CODE REVIEW

File: lib/Form/Type/FieldDefinition/FieldDefinitionType.php [hunk 1]
```diff
@@ -41,11 +44,16 @@ class FieldDefinitionType extends AbstractType
      * @var FieldsGroupsList
      */
     private $groupsList;
+    /**
```
Comment (ViniTou) [inline lib/Form/Type/FieldDefinition/FieldDefinitionType.php]: new line

**Review by ViniTou — APPROVED** [review]

**Comment by bdunogier** [comment]
I'd change the commit message to what the patch actually does, but I'm not a developer anymore ;)

`Integrated the "isThumbnail" field definition option (EZP-30882)`

File: bundle/Resources/config/services.yaml [hunk 2]
```diff
@@ -93,7 +93,10 @@ services:
 
     ezrepoforms.field_definition.form_type:
         class: "%ezrepoforms.field_definition.form_type.class%"
-        arguments: ["@ezrepoforms.field_type_form_mapper.dispatcher", "@ezpublish.api.service.field_type"]
+        arguments:
+            - "@ezrepoforms.field_type_form_mapper.dispatcher"
+            - "@ezpublish.api.service.field_type"
+            - '@eZ\Publish\Core\Repository\Strategy\ContentThumbnail\Field\ContentFieldStrategy'
```
Comment (alongosz) [inline bundle/Resources/config/services.yaml]: Please create a service in Kernel which uses interface (API/SPI?) and use it here.

**Review by alongosz — CHANGES_REQUESTED** [review]
> I'd change the commit message to what the patch actually does, but I'm not a developer anymore ;)
> 
> `Integrated the "isThumbnail" field definition option (EZP-30882)`

Definitely, I need to distinguish this on the list of changes and later on when looking for something.
Though it should rather be:
> EZP-30882: Integrated the "isThumbnail" field definition option

Other remark:

**Comment by Nattfarinn** [comment]
Repository Forms is deprecated

**Pull request closed without merging**